Source/ASDisplayNode+Yoga.mm

```diff
@@ -333,8 +337,24 @@ - (void)invalidateCalculatedYogaLayout
   }
 }
 
+- (void)semanticContentAttributeDidChange:(UISemanticContentAttribute)attribute
+{
+  if (AS_AT_LEAST_IOS9) {
+    UIUserInterfaceLayoutDirection layoutDirection =
+                         [UIView userInterfaceLayoutDirectionForSemanticContentAttribute:attribute];
+    self.style.direction = (layoutDirection == UIUserInterfaceLayoutDirectionLeftToRight
+                                             ? YGDirectionLTR : YGDirectionRTL);
+  }
```

Comment:
Nice! That class method may not be safe off-main, but I expect you've considered that. Even if so, the risk here is teeny tiny.